src/etc/inc/plugins.inc.d/loopback.inc

```diff
@@ -94,7 +94,7 @@ function loopback_configure_do($verbose = false, $device = null)
         }
     }
 
-    // destroy non existing interfaces
+    // destroy nonexistent interfaces
```

Comment:
any thoughts on nonexistent vs. non-existent ?

Reply:
Hyphens tend to die over time as new words become accepted. `email` used to have a hyphen, it's long gone as well.

That said, I'm not attached to suggestions, if you have a house style and use it consistently, I'm happy to use/enforce it.

These are just my defaults.
https://github.com/jsoref/opnsense-core/blob/spell-check/.github/actions/spelling/line_forbidden.patterns#L22

Reply:
Thanks so this as a prerequisite ... c6cb5cd7a07

Reply:
Hmm, indeed. I'll update my patterns to flag that. I have occasionally made those corrections manually, I might as well have my tool flag them for me :-).